corehq/ex-submodules/casexml/apps/case/mock.py

```diff
@@ -99,7 +99,9 @@ def __init__(self,
         """
         super(CaseBlock, self).__init__()
         self._id = case_id
-        date_modified = date_modified or datetime.utcnow()
+        now = datetime.utcnow()
+        date_modified = date_modified or now
+        date_opened = date_opened or date_modified
```

Comment:
@snopoke I think that's an oversight. we should probably only specify `date_opened` if the case is also being created.